libutils/string_sequence.c

```diff
@@ -144,18 +142,28 @@ static char *ValidDuplicate(const char *src, long n)
     return dst;
 }
 
+bool WriteLenPrefixedString(Writer *w, const char *string)
+{
+    const unsigned long str_length = strlen(string);
```

Comment:
`strlen()` returns `size_t`.

Reply:
Haha, that's your code :wink: :stuck_out_tongue: